_coffee/application.coffee

```diff
@@ -60,6 +60,10 @@ categories =
     3: 10677957 # robolectric-plugin
     4: 17395619 # gradle-android-jacoco-plugin
     5: 53143469 # ios-demos
+    6: 73317406 # gradle-build-properties-plugin
+    7: 73275282 # gradle-static-analysis-plugin
+    8: 67407288 # gradle-xcodePlugin
```

Comment:
also this seems to be a fork, I'm not sure we should include forks in these lists